<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="3,0,0">
    <property name="spacing">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="widgetTitle" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>50</height>
       </size>
      </property>
      <widget class="QLabel" name="lb_funTitle">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>10</y>
         <width>301</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>32</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>功能标签</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="lb_appstates">
       <property name="geometry">
        <rect>
         <x>520</x>
         <y>0</y>
         <width>250</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>开机自检</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="minimumSize">
       <size>
        <width>782</width>
        <height>342</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>782</width>
        <height>342</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>-1</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widgetBtns" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="pb_channelTest">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>桥路自检</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pb_channelTestIns">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>绝缘自检</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pb_voltageSet">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>电压设置</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pb_channelSelect">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>通道选择</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pb_bridgeCircuit">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>桥路</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pb_insulation">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>绝缘</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pb_cutOff">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>切断</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pb_execute">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>执行</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pb_export">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>导出记录</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pb_TEST">
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>test</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
